src/components/Callout.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <div
+    class="callout"
+    :class="[ calloutClass ]">
+    <i
+      v-if="icon"
+      class="fas"
+      :class="[calloutIcon]"></i>
+    <slot/>
+    <i v-if="closeable" class="fas fa-times callout__close"></i>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from "vue-property-decorator";
+
+@Component({})
+export default class Callout extends Vue {
+  @Prop({ default: "success" })
+  variation!: string;
+
+  @Prop({ default: true })
+  icon!: boolean;
+
+  @Prop({ default: false })
+  closeable!: boolean;
+
+  get calloutClass() {
+    return `callout--${this.variation}`;
+  }
+
+  get calloutIcon() {
+    switch (this.variation) {
+      case "success":
+      case "success-alt":
+        return "fa-check-circle";
+      case "warning":
+      case "warning-alt":
+        return "fa-times-circle";
+      case "info":
+        return "fa-info-circle";
+    }
+  }
+}
+</script>
+
+<style lang="less" scoped>
+@import "./../styles/Imports";
+
+.callout {
+  position: relative;
+  .margin-bottom(3);
+  .padding();
+  border: none;
+  .radius();
+
+  a {
+    font-weight: normal;
+    text-decoration: underline;
+    color: inherit;
+  }
+
+  .fab,
```

Comment:
let's remove this font size rule so they're just standard 14px and change to .margin-right()